l10n_it_fiscal_document_type/data/fiscal.document.type.csv

```diff
@@ -11,4 +11,15 @@ id,code,name,out_invoice,in_invoice,out_refund,in_refund,priority
 6,TD10,fattura di acquisto intracomunitario beni,False,True,False,False,3
 7,TD11,fattura di acquisto intracomunitario servizi,False,True,False,False,3
 8,TD12,documento riepilogativo,False,True,False,False,3
-13,TD20,autofattura,True,False,False,False,3
```

Comment:
I vecchi tipi che non esistono più (TD07, ..., TD12) non sono più utilizzabili (rif. https://www.fatturapa.gov.it/export/fatturazione/sdi/Specifiche_tecniche_del_formato_FatturaPA_V1.3.1.pdf, TipoDocumento).
Si potrebbe aggiungere un campo `active` e valorizzarlo con False qui nel csv in modo da archiviarli, che ne pensi?

Reply:
Esistono ancora, sono solo per la dichiarazione dati fatture (se guardi https://www.fatturapa.gov.it/export/fatturazione/sdi/Specifiche_tecniche_del_formato_FatturaPA_V1.2.2.pdf non li trovi) ma li trovi nel file Schema_DatiFattura.xsd che trovi nello zip in https://www.agenziaentrate.gov.it/portale/web/guest/provvedimento-del-28-febbraio-2020 )

Reply:
ah ho capito, quindi anche prima non servivano per la fatturazione elettronica, grazie